### ISSUE TITLE
Converting scene to tileset results in empty tileset

### ISSUE THREAD

**Issue by juicetin** [godotengine/godot#28487]
**Godot version:**
3.1.1.stable.official

**OS/device including version:**
Arch Linux, linux kernel version 5.0.4. Nvidia GTX 1060 6GB, using nvidia drivers 418.56-8 (not a rendering/graphics issue though, AFAICT)

**Issue description:**
When converting a scene to a tileset, the resulting tileset is empty - it doesn't contain anything.

**Steps to reproduce:**
I was able to reproduce using the minimal project attached to this tutorial - https://www.youtube.com/watch?v=OYn49ghh9k0&feature=youtu.be
Project link below.

After importing the project below, without doing anything else:
1. create a new scene
2. drag the `iso_tile_green.png` file onto the scene
3. save the scene
4. convert the scene to a tileset `isometric_tileset.tres`
4. double click the `isometric_tileset.tres` resource in the `Filesystem` tab - it's empty

**Minimal reproduction project:**
Project can be found here - https://github.com/kidscancode/godot_tilemap_navigation

**Comment by YeldhamDev** [comment]
The tileset was created correctly on my end.

**Comment by juicetin** [comment]
I'm on a different machine now (with the same linux versions) - and no longer seeing the issue hmm.

There was a `Node2D` at the root, and the `.png` file was dragged onto the scene, as a child node.

I'll try this again when I'm back on the machine where I first saw the issue and report back.

**Comment by KoBeWi** [comment]
>1. create a new scene
>2. drag the iso_tile_green.png file onto the scene

This will create Sprite as a root. You indeed need a Node2D root for the tileset to be correct.

**Comment by jamesmcm** [comment]
I'm having the same issue, also on Arch Linux with Godot 3.2.1.stable.custom_build (from the AUR).

Here is my tiles.tscn:

```
[gd_scene load_steps=3 format=2]

[ext_resource path="res://assets/tiles/xcom_1.png" type="Texture" id=1]

[sub_resource type="TileSet" id=1]
0/name = "grass"
0/texture = ExtResource( 1 )
0/tex_offset = Vector2( 0, 0 )
0/modulate = Color( 1, 1, 1, 1 )
0/region = Rect2( 0, 583, 32, 16 )
0/tile_mode = 0
0/occluder_offset = Vector2( 0, 0 )
0/navigation_offset = Vector2( 0, 0 )
0/shape_offset = Vector2( 0, 0 )
0/shape_transform = Transform2D( 1, 0, 0, 1, 0, 0 )
0/shape_one_way = false
0/shape_one_way_margin = 0.0
0/shapes = [  ]
0/z_index = 0

[node name="Node2D" type="Node2D"]

[node name="TileMap" type="TileMap" parent="."]
mode = 1
tile_set = SubResource( 1 )
cell_size = Vector2( 32, 16 )
format = 1
tile_data = PoolIntArray( -196588, 0, 0, -196587, 0, 0, -196586, 0, 0, -196585, 0, 0, -196584, 0, 0, -131052, 0, 0, -131051, 0, 0, -131050, 0, 0, -131049, 0, 0, -131048, 0, 0, -65516, 0, 0, -65515, 0, 0, -65514, 0, 0, -65513, 0, 0, -65512, 0, 0, 20, 0, 0, 21, 0, 0, 22, 0, 0, 23, 0, 0, 24, 0, 0, 65556, 0, 0, 65557, 0, 0, 65558, 0, 0, 65559, 0, 0, 65560, 0, 0, 131092, 0, 0, 131093, 0, 0, 131094, 0, 0, 131095, 0, 0, 131096, 0, 0, 196628, 0, 0, 196629, 0, 0, 196630, 0, 0, 196631, 0, 0, 196632, 0, 0 )
```

Here is the generated empty tiles.tres:

```
[gd_resource type="TileSet" format=2]

[resource]
```

Do I have something wrong in the original scene definition? I'm new to Godot.

**Comment by vscheff** [comment]
I'm confused on why this is closed. As of Godot 3.4.4 the option to convert a Scene to a Tileset is still a feature. Also having issues using this feature on Linux, whereas I don't run into issues on Win10.

**Comment by KoBeWi** [comment]
You can use the TileSet editor, there is one in 3.4.4 too.

**Comment by vscheff** [comment]
Yes I used the TileSet editor. Though just being able to drop all my tiles into one scene, and make them a TileSet is a much more convenient workflow. No big deal there though.

I'm still curious as to why this issue is closed with reason "The option to convert to TileSet is gone in 4.0". This option is very clearly still available as of 3.4.4. Plus, it still seems to have some issues on Linux. This issue should not be closed.
